MultiROMMgr/src/main/assets/devices.json

```diff
@@ -104,7 +104,7 @@
         },
         {
             "names": [ "yuga", "C6603", "C6602", "C6606" ],
-            "manifest_url": "http://xperiamultirom.github.io/manifests/yuga/manifest.json",
+            "manifest_url": "http://team-simplicit.com/multirom/yuga/manifest.json",
```

Comment:
Was this change intentional? If so it seems unrelated to this pull request and should be in a separate PR of its own.